core/src/i18n/en.yaml

```
@@ -35,19 +60,25 @@ common:
   created: Created
   delete-dialog:
     delete-confirm: Don't ask this again
+    information: You've switched off delete confirmations for all resources. You can switch them back on in Preferences.
```

```suggestion
    information: You've switched off removal confirmations for all resources. You can switch them back on in Preferences.
```

As the dialog tile is "Remove..." we should keep using the word "remove".